### PR TITLE
Fix styling

### PR DESCRIPTION
**Pull request by FrankApiyo**
This PR contains css written by @oluwatobim

### CODE REVIEW

**Comment by FrankApiyo** [comment]
todo:

- [ ] rename css classes to ensure they adhere to **BEM**.

File: src/components/BasicInformation/index.css [hunk 1]
```diff
@@ -1,58 +1,59 @@
 .dataItem {
-  border: 1px solid black;
-  width: 20rem;
+    border: 1px solid black;
+    width: 20rem;
```
Comment (cKellyDesign) [inline src/components/BasicInformation/index.css]: Let's keep the `.css` tabbing to 2 spaces.

File: src/containers/pages/Home/index.css [hunk 2]
```diff
@@ -15,18 +15,20 @@
   max-width: 25%;
   min-width: 140px;
   flex: 1;
-  font-family: 'Trebuchet MS', 'Helvetica';
+  font-family: 'Roboto', 'Helvetica';
```
Comment (cKellyDesign) [inline src/containers/pages/Home/index.css]: 🙏 I've never ever liked Trebuchet very much, Roboto is much better!

File: src/components/VillageData/index.css [hunk 3]
```diff
@@ -7,9 +7,9 @@
   width: 100%;
 }
 
-.villageDataRow {
+/* .villageDataRow {
   margin-top: 3% !important;
-}
+} */
```
Comment (cKellyDesign) [inline src/components/VillageData/index.css]: Shall we simply remove this?

File: src/components/BasicInformation/index.css (hunk 1 above)
Comment (FrankApiyo) [reply src/components/BasicInformation/index.css]: done!

File: src/components/VillageData/index.css (hunk 3 above)
Comment (FrankApiyo) [reply src/components/VillageData/index.css]: removed!

**Comment by FrankApiyo** [comment]
> todo:
> 
>     * [ ]  rename css classes to ensure they adhere to **BEM**; more on this is detailed here point_right #139

@cKellyDesign, This may not be achieved in this PR, but there's an issue for it.

**Comment by moshthepitt** [comment]
@FrankApiyo can we get screenshots?

**Comment by FrankApiyo** [comment]
Screenshot from 2020-01-16 14-37-40

**Comment by FrankApiyo** [comment]
Screenshot from 2020-01-16 14-37-17

**Comment by FrankApiyo** [comment]
Screenshot from 2020-01-16 14-37-12

**Pull request merged**